README.md

```diff
@@ -15,6 +15,7 @@ Here's a list of all available options:
  - **--port**: when the web server isn't running on port 80.
  - **--wordlist**: specify an alternative location for the wordlist.
  - **--ssl**: `on` or `off` depending on whether you want to connect with SSL.
+ - **--output**: Optionally specify an alternative file to write the output to. (Default, current directory).
```

Comment:
Lowercase optionally and replace `(Default, current directory)` with Defaults to `output.txt` in the current directory. Thanks!